gradle/scripts/yaml.gradle

```diff
@@ -44,6 +63,31 @@ def validateSchema(yamlFile, jsonSchemaFile) {
     }
 }
 
+ext.validateMapsYamlLinks = { yamlFile ->
+    def yaml = new Yaml()
+    def maps = yaml.load(yamlFile.text)
+    def imageUrls = maps . collect { it['img'] } . findAll { it != null }
+
+    def clientConfig = new DefaultAsyncHttpClientConfig.Builder()
+            .setFollowRedirect(true)
+            .build()
+    def client = new DefaultAsyncHttpClient(clientConfig)
```

Comment:
This HTTP client executes multiple requests in parallel backed by a thread pool.